File: models/cliente.py
"""Cliente — nome e indicação de crianças (RF05)."""


class Cliente:
    def __init__(self, nome: str, possui_criancas: bool):
        pass

    def get_nome(self) -> str:
        raise NotImplementedError

    def get_possui_criancas(self) -> bool:
        raise NotImplementedError

File: src/models/Cliente.ts
/** Cliente — nome e indicação de crianças (RF05). */
export class Cliente {
  private nome: string;
  private possuiCriancas: boolean;

  constructor(params: { nome: string; possuiCriancas: boolean }) {
    this.nome = params.nome;
    this.possuiCriancas = params.possuiCriancas;
  }

  getNome(): string {
    return this.nome;
  }

  getPossuiCriancas(): boolean {
    return this.possuiCriancas;
  }
}
